.github/workflows/ci.yml

```diff
@@ -13,6 +13,7 @@ on:
 env:
   RUSTFLAGS: -Dwarnings
   RUST_BACKTRACE: 1
+  minrust: 1.42
```

Comment:
IIRC, our MSRV is 1.51: https://github.com/tokio-rs/valuable/pull/11#discussion_r630461466